controllers/users/users.js

```diff
@@ -33,6 +70,7 @@ exports.registerUser = (req, res) => {
           if (err) throw err;
           // Store hash in your password DB.
           newUser.password = hash;
+          newUser.activationKey = Math.floor(Math.random()*10000000);
```

Comment:
We can use bcryptjs and generate some hash from user details and used it as activationKey.
If we would like to transport some additional information with activationKey itself and not to depend on database support we can use signed or encrypted JWT token.